deploy.spryker-b2c-eu.yml

```diff
@@ -0,0 +1,210 @@
+version: "0.1"
```

Comment:
I do not think we need this and all of the new deploy.yml files below. This must be used for development purposes (for staging deploys etc). Let's not introduce new deploy.ymls we are not sure we are using.